### PR TITLE
Disable devel repo after installing vbox fencing

### PR DESCRIPTION
**Pull request by jgrund**
Signed-off-by: Joe Grund <jgrund@whamcloud.io>

### CODE REVIEW

File: iml-sandbox/Vagrantfile [hunk 1]
```diff
@@ -439,6 +439,7 @@ def provision_fence_agents(config)
     yum install -y yum-plugin-copr
     yum -y copr enable managerforlustre/manager-for-lustre-devel
```
Comment (johnsonw) [inline iml-sandbox/Vagrantfile]: Do we still need this line?

Comment (jgrund) [reply iml-sandbox/Vagrantfile]: yes

Comment (jgrund) [reply iml-sandbox/Vagrantfile]: It's what allows the fence-vbox package to be available.

Comment (johnsonw) [reply iml-sandbox/Vagrantfile]: ok

**Pull request merged**